### PR TITLE
fix: Fixes Github Releases

### PR DESCRIPTION
**Pull request by ElvenSpellmaker**
# Pull Request Template

## 📲 What

- Fixes Github Releases, last release was Nov 2022
   - Uses the correct user `amidostacks`
   - Allows pre-releases to be released under branches
   - Release job now moves dependencies based on PR or `master`
- Bumps patch version of cluster to satisfy tests
- Updates `Deploy-HelmCharts` with latest version (NOTE: This should go into IR)

## 🤔 Why

Releases are broken since 1st Dec, 2022

### CODE REVIEW

**Review by russellseymour — APPROVED** [review]
:+1:

File: build/scripts/Deploy-Charts.ps1 [hunk 1]
```diff
@@ -4,36 +4,36 @@ param (
 
     [string]
     # Path to the configuration file stating which charts should be deployed
-    $path = $env:HELM_CHARTS,
+    $Path = $env:HELM_CHARTS,
```
[the remark this answers is not in the document]
Comment (ElvenSpellmaker) [reply build/scripts/Deploy-Charts.ps1:7]: As we discussed it's a pwsh stylistic thing, incoming script Parameters are upper cased. Article if you're interested: https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.4#use-pascal-case-for-parameter-names

**Pull request merged**